tests/generative/data_setup.py

```diff
@@ -7,59 +6,43 @@
     SelectPatientTable,
     SelectTable,
 )
-from databuilder.sqlalchemy_types import Integer, type_from_python_type
 
-from ..lib.util import next_id, null
+from ..lib.util import orm_class_from_schema
 
 
 def setup(schema, num_patient_tables, num_event_tables):
-    registry = sqlalchemy.orm.registry()
-    patient_id_column = "patient_id"
+    base_class = sqlalchemy.orm.declarative_base()
 
     patient_table_names, patient_classes = _build_orm_classes(
-        "p", num_patient_tables, schema, patient_id_column, registry
+        "p", num_patient_tables, schema, base_class
     )
     event_table_names, event_classes = _build_orm_classes(
-        "e", num_event_tables, schema, patient_id_column, registry
+        "e", num_event_tables, schema, base_class
     )
 
     all_patients_query = _build_query(patient_table_names, event_table_names, schema)
 
     return (
-        patient_id_column,
+        # We don't currently support different names for this column
+        "patient_id",
```

Comment:
Don't we? Isn't this the backend patient join column?

Reply:
Ah yes, there's a useful clarification to made here. We support changing this in the SQL schema (for the `patient_join_column` attribute as you say) but in the logical schema it's always called `patient_id`. And where we're generating ORM classes from the logical schema we're implicitly guaranteeing that the two match.

I think it would be better actually to refactor this code so we don't return this value and just have it set as a module level constant. I'll do that now.